packages/monorepo-tools/package.json

```diff
@@ -13,6 +13,7 @@
   "scripts": {},
   "dependencies": {
     "@yarnpkg/lockfile": "^1.0.0",
+    "babel-settings": "0.0.1",
```

Comment:
devDependencies?

Reply:
babel-settings is more complicated, babel-runtime is a runtime dependency rather than a dev dependency.

Reply:
hmm... good point though.. let me reconsider this.